input/pagecontent/changes.md

```diff
@@ -42,7 +43,7 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from MedicationRequest.dispenseRequest, MedicationRequest.dispenseRequest.validityPeriod, MedicationRequest.dispenseRequest.numberOfRepeatsAllowed, MedicationRequest.dispenseRequest.quantity [FHIR-45088](https://jira.hl7.org/browse/FHIR-45088)
   - removed Must Support from MedicationRequest.substitution, MedicationRequest.substitution.allowed[x] [FHIR-45088](https://jira.hl7.org/browse/FHIR-45088)
   - added Must Support to MedicationRequest.reasonReference [FHIR-45090](https://jira.hl7.org/browse/FHIR-45090)
-  - changed medicationCodeableConcept.coding slicing rule from slicing by system to slice by value set [FHIR-46391](https://jira.hl7.org/browse/FHIR-46391)
+  - support for Australian Medication (AMT) codes and PBS Item Codes in MedicationRequest.medicationCodeableConcept changed to define slicing in AU Core (no longer inherited from AU Base as per [FHIR-44821](https://jira.hl7.org/browse/FHIR-44821)). Slicing discriminator changed from slicing by value:system to slicing by value set and value set binding is required within the AMT and PBS slices. [FHIR-46391](https://jira.hl7.org/browse/FHIR-46391)
```

Comment:
Australian Medication should read  Australian Medicines Terminology but recommend to just say "AMT" in place of "Australian Medication (AMT)" - this comment applies to two locations in this file.

Reply:
Fixed